build.sbt

```diff
@@ -28,6 +28,8 @@ lazy val commonSettings = Seq(
       "-unchecked"),
   libraryDependencies ++= commonDeps)
 
+publish / skip := true
```

Comment:
I'm always confused... this setting here applies just to the top-level project, or to **all** (sub)projects, including the top-level one?

Reply:
The top level one

Reply:
Ok... so its `ThisBuild / ...` that applies to all (sub)projects? or `Global / ...`? 🤔